build/openwisp_openvpn/revokelist.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Since you are using checksums to validate if configs have changed would it be more maintainable to combine revokelist.sh and openvpn.sh?  Just have a single script that runs via cron to perform routine maintenance and configuration update?

Reply:
I kept them in different scripts because I assume that the frequency of updating revoke list would much less, because after running the revoke list script, openvpn is restarted, so revoke list is updated only at 3AM on Sundays, while openvpn config is checked every minute because I assume the user will want to see the updated settings as soon as possible! 
Is my understanding about the frequiency of scripts correct? :smile:

Reply:
I assumed you had a good answer :smile:  Thanks!

Reply:
Is openvpn required to restart just from checking for revocations or only when it needs to update it's list of revoked certificates?  It seems like it should check daily and only restart as necessary.

Reply:
Okay, I am not very sure about how CRLs work, but everytime I download the CRL, I get a new file with different SHA1, so I assumed that that's expected nature of CRL file. Hence I can't know for sure if there is a change in revokation list and therefore I need to restart OpenVPN even after just downloading the list (with optentially no change.)

i.e. If I download `http://dashboard.openwisp.org/x509/ca/1.crl` and then download the same again, I get different values of the crl file.